app/filter.py

```diff
@@ -363,6 +363,9 @@ def view_image(self, soup) -> BeautifulSoup:
 
         # get some tags that are unchanged between mobile and pc versions
         search_input = soup.find_all('td', attrs={'class': "O4cRJf"})[0]
+
+        search_input = BeautifulSoup(re.sub(r'(\s)-site:[a-zA-Z0-9_\.]+', '',
+                                     str(search_input)), 'lxml')
```

Comment:
Should use a different parser here -- iirc `lxml` isn't included by default.

```suggestion
                                     str(search_input)), 'html.parser')
```

Reply:
Weird, it works for me in my virtualenv